### PR TITLE
chore: cleanup branching modal

### PR DESCRIPTION
**Pull request by loong**
Changes:
 * ~Remove PITR notice~, decided to not remove for now
 * Make icons less warny and less anxiety-inducing
 * Make copy more concise

Before:
Screenshot 2024-08-14 15 35 09@2x

After:
Screenshot 2024-08-14 16 56 00@2x

### CODE REVIEW

File: apps/studio/components/layouts/AppLayout/EnableBranchingButton/EnableBranchingModal.tsx [hunk 1]
```diff
@@ -211,36 +209,33 @@ const EnableBranchingModal = () => {
...
-                      <figure className="w-10 h-10 rounded-md bg-warning-200 border border-warning-400 flex items-center justify-center">
-                        <FileText className="text-warning" size={20} strokeWidth={2} />
+                      <figure className="w-10 h-10 rounded-md bg-info-200 border border-info-400 flex items-center justify-center">
+                        <FileText className="text-info" size={20} strokeWidth={2} />
                       </figure>
                     </div>
                     <div className="flex flex-col gap-y-1">
                       <p className="text-sm text-foreground">
-                        Branching uses your GitHub repository to apply migrations
+                        Migrations are apllied from your GitHub repository
```
Comment (alaister) [inline apps/studio/components/layouts/AppLayout/EnableBranchingButton/EnableBranchingModal.tsx:233]:
```suggestion
                        Migrations are applied from your GitHub repository.
```

**Review by alaister — APPROVED** [review]
LGTM!

- Viewed on preview

**Comment by loong** [comment]
reverted and the PITR notice stays for now

Screenshot 2024-08-15 17 14 35@2x

**Review by joshenlim — APPROVED** [review]
LGTM! Checked on the staging preview the copy and icon color changes

**Pull request merged**